ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -113,8 +127,58 @@ export interface IconConfig {
   hasContent: boolean;
 }
 
+export interface StatusIconProps {
+  iconPath: string;
+  iconConfig: IconConfig;
+  config: ConfigResponse;
+  children: React.ReactNode;
+}
+
+export const StatusIcon = ({
+  iconPath,
+  iconConfig,
+  config,
+  children,
+}: StatusIconProps) => {
+  const iconStyle: CSSProperties = showAppsPanel(config)
+    ? { width: '36px', position: 'absolute' }
+    : { width: '22px', position: 'absolute' };
```

Comment:
Instead of passing `config` throughout the components, `useConfig` can be moved to this component instead.

Reply:
Good idea